apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1809,8 +1859,8 @@ send_deposit_created_msg(SignedTx, Updates,
     aesc_session_noise:deposit_created(Sn, Msg),
     log(snd, ?DEP_CREATED, Msg, Data).
 
-check_deposit_created_msg(_Msg, #data{ client_connected = false }) ->
-    {error, client_disconnected};
+%% check_deposit_created_msg(_Msg, #data{ client_connected = false }) ->
+%%     {error, client_disconnected};
```

Comment:
```suggestion
```

Reply:
Code deleted because the check is now done in the `request_signing()` function (and the operation potentially allowed if the request is already co-signed.)